src/handlers/scheduledEventHandler.ts

```diff
@@ -1,8 +1,63 @@
+import { KVNamespace } from '@cloudflare/workers-types';
+
 import { handleConfig } from '../config/config';
-import { env } from '../types/global.types';
+import { NAMESPACE_NAME } from '../constants';
+import { env, NicknameUpdateResponseType } from '../types/global.types';
+import { generateJwt } from '../utils/generateJwt';
 
 export async function ping(env: env) {
 	const url = handleConfig(env);
 	const response = await fetch(`${url.baseUrl}/healthcheck`);
 	return response;
 }
+
+export async function callDiscordNicknameBatchUpdate(env: env) {
+	const namespace = env[NAMESPACE_NAME] as unknown as KVNamespace;
+	let lastNicknameUpdate: string | null = '0';
+	try {
+		lastNicknameUpdate = await namespace.get('DISCORD_NICKNAME_CHANGED');
```

Comment:
how about using "NICKNAME_CHANGE_TIMESTAMP" here

Reply:
@ankushdharkar please suggest if this name needs to be changed.

Reply:
`DISCORD_NICKNAME_CHANGED` sounds like a boolean to me.

What about `DISCORD_NICKNAME_UPDATED_TIME`?

Reply:
To update this, we should update the KV name
Will be tackling this in a separate PR, created a issues for it https://github.com/Real-Dev-Squad/cron-jobs/issues/16

Reply:
Updated it https://github.com/Real-Dev-Squad/cron-jobs/issues/12#issuecomment-1793027414